README.md

```diff
@@ -40,6 +40,10 @@ Jaeger is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
   * [Kubernetes templates](https://github.com/jaegertracing/jaeger-kubernetes)
   * [OpenShift templates](https://github.com/jaegertracing/jaeger-openshift)
 
+The Jaeger contributors meet bi-weekly, and everyone is welcome to join.
```

Comment:
I would add a header "Project Status Bi-Weekly Meeting"

Reply:
Also, this is an odd location, it's technically under "## Related Repositories". I would move it above that, or down under the Contribute section.

Reply:
+1 , I actually meant to add in the "Contribute" section :)